### ISSUE TITLE
Windows Terminal doesn't display Arabic Characters

### ISSUE THREAD

**Issue by Dev-mdz** [microsoft/terminal#4122]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18363.535]
Windows Terminal version (if applicable): 0.7.3451.0

Any other software?
```

# Steps to reproduce

Opening Windows Terminal and connecting via SSH to the server (**CentOS release 6.10 (Final)**), doesn't really matter if you use the SSH step but I didn't test. then `git diff` a certain file that contains Arabic characters (I didn't try this with other languages like Mandarin Chinese or Hebrew to see if the error exists).

# Expected behavior

After running `git diff` on a file that contains Arabic text, it should display the correct characters.
It works fine when I use the command `cat`.

# Actual behavior

Instead it display the following:
The text that looks like `<D8><A7>`... represents the arabic text.

![Windows_Terminal](https://user-images.githubusercontent.com/59362525/71817151-41be5e00-3085-11ea-9cec-1f75f34ca37c.PNG)

**Comment by zadjii-msft** [comment]
To be fair, I think that the translation of Arabic characters to `<D8><A7>...` is being done by `git diff`, not the Terminal.

That being said, we definitely _don't_ support Arabic or any RTL text currently, which I think is being tracked in #538

**Comment by Dev-mdz** [comment]
I tested Mandarin Chinese, Hebrew and accented French letters like accèéàüûùÔôö etc, and I got the same result, characters are not displayed.

I tried with a different SSH client ([Termius](https://termius.com/)) and it displays the characters, however they're backwards.

Anyway, there are more bigger issues that need to be solved than this. The key takeaway from this is that the French accents don't display correctly.
